docs/using/tutorial/moving-volume-tutorial.rst

```diff
@@ -0,0 +1,192 @@
+To install the Flocker client, run the following:
+
+OS X
+----
+
+.. task:: test_homebrew flocker-|latest-installable|
+   :prompt: you@laptop:~$
+
+Ubuntu 14.04
+------------
+
+.. task:: install_cli ubuntu-14.04
+   :prompt: you@laptop:~$
+
+Ubuntu 15.04
+------------
+
+.. task:: install_cli ubuntu-15.04
+   :prompt: you@laptop:~$
+
+Installing Flocker on Local VMs
+===============================
+
+In this step, you will install two instances of Flocker, each on a separate host.
+Flocker manages the links, ports, and volumes associated with Docker containers and can move them around after deployment.
+
+#. Download the Vagrant configuration file, and the cluster and user credentials:
+
+   .. version-code-block:: console
+
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.crt
+      you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.key
+
+#. Use ``vagrant up`` to start and provision the VMs:
+
+   .. prompt:: bash you@laptop:~$
+
+      vagrant up
+
+Deploying an Application on the First Host
+==========================================
+
+You will now have the client installed on your local machine, and two instances of Flocker, each on a different host.
+The next step is to create two Docker containers on one of the hosts.
+One container has a Python web application and the other has a Redis server, which stores its data on a volume.
+
+Now you can try our simple tutorial: a Python web application and a Redis server.
+Before you begin, you'll need to download our sample ``YAML`` files:
+
+.. container:: hidden
+
+   .. Create the files to be downloaded with ``cURL``, but don't show download links for them.
+
+   :download:`docker-compose.yml`
+   :download:`deployment-node1.yml`
+   :download:`deployment-node2.yml`
+		 
+.. version-code-block:: console
+
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/docker-compose.yml
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node1.yml
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node2.yml
+
+.. XXX The version-code-block presents a different style to prompts, so this document would benefit from FLOC 2141 (version-prompt)
+
+The :file:`docker-compose.yml` file describes your distributed application:
+
+.. literalinclude:: docker-compose.yml
+   :language: yaml
+
+The :file:`deployment-node1.yml` file describes which containers to deploy, and where:
+
+.. literalinclude:: deployment-node1.yml
+   :language: yaml
+
+#. Use the Flocker CLI to deploy both your web application and server onto one of the virtual machines that you have just created:
```

Comment:
It's annoying that these appear indented because of the styling when they render as a numbered list. The indentation makes it easy to miss them.